workbench-core/datasets/src/schemas/createDataSet.ts

```diff
@@ -12,10 +12,18 @@ const CreateDataSetSchema: Schema = {
   properties: {
     datasetName: { type: 'string' },
     path: { type: 'string' },
-    region: { type: 'string' }
+    description: { type: 'string' },
+    customMetadata: {
```

Comment:
Are we expecting other metadata to be provided during the create call? Any downside in replacing `customMetadata` object with a string attribute `owner`?

Reply:
Done, but property name is owningProjectId